frontend/src/components/worksheets/items/ActionButtons.js

```diff
@@ -133,9 +136,23 @@ class ActionButtons extends React.Component<{
                 {showBundleOperationButtons ? (
                     <BundleBulkActionMenu
                         handleSelectedBundleCommand={handleSelectedBundleCommand}
-                        togglePopup={togglePopup}
+                        toggleCmdDialog={toggleCmdDialog}
+                        toggleCmdDialogNoEvent={toggleCmdDialogNoEvent}
                     />
                 ) : null}
+                <Tooltip title='Paste text or uuids to this worksheet'>
+                    <Button
+                        size='small'
+                        color='inherit'
+                        aria-label='Paste'
+                        onClick={toggleCmdDialog('paste')}
+                        disabled={!editPermission}
+                        id='paste-button'
+                    >
+                        <NoteAddIcon className={classes.buttonIcon} />
+                        PasteBoard
```

Comment:
Can we just call it "Paste"? "Pasteboard" sounds kind of unfamiliar / confusing.

```suggestion
                        Paste
```